Dockerfile.mysql

```diff
@@ -0,0 +1,13 @@
+FROM mysql:latest
+
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD=root
+
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
+# Copy the MySQL initialization scripts
+COPY init.sql /docker-entrypoint-initdb.d/
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/e8fa9ee2-7133-4b24-9529-1f9bd42d5652)
Add Eof(End-of-file), need to add an empty line at the end